### PR TITLE
Remove All Hardcoded strings and other warnings

### LINKED ISSUE #1090: Remove all "Hardcoded Strings" in all XML file of this app.
**Summary:**

Remove all "Hardcoded string" by creating the strings in "strings.xml" file.

**Steps to reproduce:**

Hardcoded strings are present in "item_si.xml" and many more XML files. Simply open the file and wait for the loading of XML. You will find that there are many hardcoded strings.

**Expected behavior:**

The app should not show the "Hardcoded string" warning.

**Observed behavior:**

Warning Screenshot in "item_si" XML file:
Capture

### PR DESCRIPTION
**Pull request by deepak-prajapatii**
## Issue Fix
Fixes #1090 

## Description
Removes All hardcoded strings and other warnings in this app.

##

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Fix the warning of showing hardcoded strings and other warnings too.

### CODE REVIEW

**Comment by deepak-prajapatii** [comment]
@Nayan @vorburger @vishwasbabu @siebrand Please review my PR ASAP.

**Comment by deepak-prajapatii** [comment]
@Nayan @vorburger @vishwasbabu @siebrand Please review my PR ASAP.

**Comment by EGOR-IND** [comment]
Seems fine to me, you pretty much removed many warnings but why did you changed gradle dependency?

**Comment by vorburger** [comment]
Hello. Thank you for pinging me here. I unfortunately don't have the time
to contribute to this particular sub-project, and therefore regret that I
cannot help review PRs in this repo. Thanks for your understanding!

**Comment by deepak-prajapatii** [comment]
> Seems fine to me, you pretty much removed many warnings but why did you changed gradle dependency?

Okay, I will undo the changes in Gradle dependency.

**Comment by deepak-prajapatii** [comment]
@EGOR-IND @vorburger @Nayan @siebrand Please review my PR again.

**Comment by deepak-prajapatii** [comment]
@EGOR-IND @vorburger @Nayan @siebrand Please review my PR again.

**Comment by vorburger** [comment]
@deepak-prajapatii I unfortunately don't have the time to contribute to this particular sub-project, and therefore regret that I cannot help review PRs in this repo. Thanks for your understanding!

**Comment by deepak-prajapatii** [comment]
> @deepak-prajapatii I unfortunately don't have the time to contribute to this particular sub-project, and therefore regret that I cannot help review PRs in this repo. Thanks for your understanding!

Okay Sir.

**Comment by deepak-prajapatii** [comment]
@EGOR-IND @Nayan @siebrand Please review my PR again.

**Comment by deepak-prajapatii** [comment]
@EGOR-IND @Nayan @siebrand Please review my PR again.

**Comment by deepak-prajapatii** [comment]
@EGOR-IND @Nayan @siebrand Please review my PR again.

**Comment by deepak-prajapatii** [comment]
@EGOR-IND @siebrand Please review my PR.

**Comment by EGOR-IND** [comment]
@deepak-prajapatii let me see what you did.

File: build.gradle [hunk 1]
```diff
@@ -8,7 +8,7 @@ buildscript {
         maven { url 'https://maven.google.com' }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.6.3'
+        classpath 'com.android.tools.build:gradle:4.1.1'
```
Comment (EGOR-IND) [inline build.gradle]: Yeah everything is fine but why is this here, i mean why you changed gradle

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,7 +1,6 @@
-#Mon Jan 14 18:31:05 IST 2019
+#Thu Jan 07 20:26:35 IST 2021
```
Comment (EGOR-IND) [inline gradle/wrapper/gradle-wrapper.properties]: And what's the use of this one.

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,7 +1,6 @@
...
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.4.1-all.zip
-
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-all.zip
```
Comment (EGOR-IND) [inline gradle/wrapper/gradle-wrapper.properties]: And this one tooo what is this used for, can you explain all these 3 changes that i commented on.

File: build.gradle (hunk 1 above)
Comment (deepak-prajapatii) [reply build.gradle]: Okay I undo the changes in few minutes

**Review by EGOR-IND — CHANGES_REQUESTED** [review]
Can you please squash all your commits in one commit

**Comment by deepak-prajapatii** [comment]
@EGOR-IND I will undo the changes as soon as possible. I am a fresher and don't know so much about the Gradle dependencies.

**Comment by EGOR-IND** [comment]
> @EGOR-IND I will undo the changes as soon as possible. I am a fresher and don't know so much about the Gradle dependencies.

Did you updated the gradle plugins when you started making changes.

**Comment by deepak-prajapatii** [comment]
> > @EGOR-IND I will undo the changes as soon as possible. I am a fresher and don't know so much about the Gradle dependencies.
> 
> Did you updated the gradle plugins when you started making changes.

Yeah, I thought it was a studio update

**Comment by deepak-prajapatii** [comment]
@EGOR-IND Hey Please check again I have undo the changes. Please if there is something wrong, guide me.

**Comment by EGOR-IND** [comment]
> > > @EGOR-IND I will undo the changes as soon as possible. I am a fresher and don't know so much about the Gradle dependencies.
> > 
> > 
> > Did you updated the gradle plugins when you started making changes.
> 
> Yeah, I thought it was a studio update

You should not have done that it creates error, go look at some java file you will find alots of errors there i think you should delete this project and clone it again and then without updating you should start contributing.

**Comment by deepak-prajapatii** [comment]
> > > > @EGOR-IND I will undo the changes as soon as possible. I am a fresher and don't know so much about the Gradle dependencies.
> > > 
> > > 
> > > Did you updated the gradle plugins when you started making changes.
> > 
> > 
> > Yeah, I thought it was a studio update
> 
> You should not have done that it creates error, go look at some java file you will find alots of errors there i think you should delete this project and clone it again and then without updating you should start contributing.

okay
